JavascriptMornings/Maitee
# Add simple server to project

## Review checklist


## Testing

Steps to test the change:
1. Download Branch
2. Run `npm start` 
3. Browse http://127.0.0.1:8081/ 
4. see if index.html is working correctly 
## Additional notes

## Deploy checklist
